src/shim/shim-config.ts

```diff
@@ -377,6 +362,9 @@ class Config implements IConfig {
         ) {
             Logger.getInstance().warn("The webInstrumentation configuration options are not supported by the shim.");
         }
+        if (this.correlationHeaderExcludedDomains) {
```

Comment:
Why is this deprecated? I know incorrect behavior was present but is not possible to control which requests to propagate the header?

Reply:
Removed deprecation and created a task to track this.